src/apps/dictionaries/components/DictionaryForm.tsx

```diff
@@ -215,34 +216,44 @@ const DictionaryForm: React.FC<Props> = ({
                 <ErrorMessage name="preferred_source" component="span" />
               </Typography>
             </FormControl>
-            <FormControl
-              fullWidth
-              // required
-              margin="normal"
-            >
-              <InputLabel htmlFor="owner_url">Owner</InputLabel>
-              <Field
-                value=""
-                disabled={editing || isSubmitting}
-                name="owner_url"
-                id="owner_url"
-                multiline
-                rowsMax={4}
-                component={Select}
-              >
-                {showUserName(profile)}
-                {showOrganisationHeader(usersOrgs)}
-                {showUserOrganisations(usersOrgs)}
-              </Field>
-              <Typography color="error" variant="caption" component="div">
-                <ErrorMessage name="owner_url" component="span" />
-              </Typography>
-            </FormControl>
-            <FormControl
-              fullWidth
-              // required
-              margin="normal"
-            >
+            {viewing || editing ? (
+              <FormControl fullWidth margin="normal">
+                <Field
+                  defaultValue={getOrg || "Owner"}
+                  label="Owner"
+                  disabled={editing || isSubmitting}
+                  name="owner_url"
+                  id="owner_url"
+                  multiline
+                  rowsMax={4}
+                  component={MuiTextField}
+                />
+                <Typography color="error" variant="caption" component="div">
+                  <ErrorMessage name="owner_url" component="span" />
+                </Typography>
+              </FormControl>
+            ) : (
+              <FormControl fullWidth margin="normal">
+                <InputLabel htmlFor="owner_url">Owner</InputLabel>
+                <Field
+                  value=""
+                  disabled={editing || isSubmitting}
```

Comment:
```suggestion
                  disabled={isSubmitting}
```

Reply:
We would want the owner field not to be editable in edit view